packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m

```diff
@@ -417,70 +406,72 @@ - (void)updateAutoMediaPlaybackPolicy:(NSNumber*)policy
   }
 }
 
-- (bool)loadRequest:(NSDictionary<NSString*, id>*)request {
+- (bool)loadRequest:(WKWebView *)webView request:(NSDictionary<NSString *, id> *)request {
   if (!request) {
     return false;
   }
 
-  NSString* url = request[@"url"];
+  NSString *url = request[@"url"];
   if ([url isKindOfClass:[NSString class]]) {
     id headers = request[@"headers"];
     if ([headers isKindOfClass:[NSDictionary class]]) {
-      return [self loadUrl:url withHeaders:headers];
+      return [self loadUrl:webView url:url withHeaders:headers];
     } else {
-      return [self loadUrl:url];
+      return [self loadUrl:webView url:url];
     }
   }
 
   return false;
 }
 
-- (bool)loadUrl:(NSString*)url {
-  return [self loadUrl:url withHeaders:[NSMutableDictionary dictionary]];
+- (bool)loadUrl:(WKWebView *)webView url:(NSString *)url {
```

Comment:
When updating the `(bool)loadUrl:(WKWebView *)webView url:(NSString *)url withHeaders:(NSDictionary<NSString *, NSString *> *)headers` method to be static, this method could also be marked static.